boot/boot.js

```diff
@@ -1444,6 +1444,7 @@ $tw.loadTiddlersBrowser = function() {
 	for(var t=0; t<containerIds.length; t++) {
 		$tw.wiki.addTiddlers($tw.wiki.deserializeTiddlers("(DOM)",document.getElementById(containerIds[t])));
 	}
+	callback();
```

Comment:
>But this change will allow me to write it to be async in my own software if I want to, without having to redistribute a modified version of tiddlywiki along with my own code.

I need more context this does not sound correct. You can call sync code in your own async flow. The extra callback buys you nothing in this case. Are you using some kind of meta-programming that requires all functions have a callback?